nixos/modules/services/network-filesystems/ipfs.nix

```diff
@@ -0,0 +1,118 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  inherit (pkgs) ipfs;
+
+  cfg = config.services.ipfs;
+
+
+  ipfsFlags = ''${if cfg.autoMigrate then "--migrate" else ""}
+                ${if cfg.enableGC    then "--enable-gc" else ""}
+                ${toString cfg.extraFlags}'';
```

Comment:
Does not this add newlines to `ipfsFlags`, which breaks the service below?
`${optionalString cfg.autoMigrate "--migrate"}` could be used here.